translate/convert/test_php2po.py

```diff
@@ -58,6 +60,34 @@ def test_convertphp(self):
         assert pounit.source == "entry"
         assert pounit.target == ""
 
+    def test_convertphptemplate(self):
+        """checks that the convertphp function is working with template"""
+        phpsource = """$_LANG['simple'] = 'entry';"""
+        phptemplate = '''$_LANG['simple'] = 'source';'''
+        posource = self.convertphp(phpsource, phptemplate)
+        pofile = po.pofile(wStringIO.StringIO(posource))
+        pounit = self.singleelement(pofile)
+        assert pounit.source == "source"
+        assert pounit.target == "entry"
+
+    def test_convertphpmissing(self):
+        """checks that the convertphp function is working with missing key"""
+        phpsource = """$_LANG['simple'] = 'entry';"""
+        phptemplate = '''$_LANG['missing'] = 'source';'''
+        posource = self.convertphp(phpsource, phptemplate)
+        pofile = po.pofile(wStringIO.StringIO(posource))
+        pounit = self.singleelement(pofile)
+        assert pounit.source == "source"
+        assert pounit.target == ""
+
+    def test_convertphpempty(self):
+        """checks that the convertphp function is working with empty template"""
+        phpsource = """"""
+        phptemplate = ''''''
```

Comment:
Nit: In this case I would just go for the simplest way to specify a blank string: `""`